src/main/java/pl/wojciechkostecki/revel/security/JwtTokenUtil.java

```diff
@@ -0,0 +1,68 @@
+package pl.wojciechkostecki.revel.security;
+
+import io.jsonwebtoken.*;
+import lombok.RequiredArgsConstructor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Component;
+
+import java.util.Date;
+
+import static java.lang.String.format;
+
+
+@Component
+@RequiredArgsConstructor
+public class JwtTokenUtil {
+    private final String jwtSecret = "zdtlD3JK56m6wTTgsNFhqzjqP";
+    private final String jwtIssuer = "wojciechkostecki.pl";
+
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    public String generateAccessToken(UserDetails user) {
```

Comment:
Same here, what is jwt, how is it built, etc. Most of the stuff about JWT sine they might ask you about this on your interview.

Reply:
Ok, thanks for the hint.